docs/Product-Overview.md

```diff
@@ -0,0 +1,72 @@
+# Product Overview
+
+## OpenTripPlanner project
+
+OpenTripPlanner is a group of open source software applications that help individuals and organizations 
+identify and digitize multimodal trip plans based on OpenStreetMap (OSM) and other standardized data 
+sources (e.g. GTFS, GBFS, NeTEx).
+
+OpenTripPlanner is a community of dozens of individuals and organizations that work collaboratively to 
+improve multimodal trip planning best practices and to make it easier for public transit agencies and 
+public transit riders to publish and access information about transit services.
+
+OpenTripPlanner is locally managed in as many different ways as there are OpenTripPlanners projects. 
+OpenTripPlanner consistently and dependably delivers multimodal trip plans to millions or riders 
+everyday in dozens of countries around the globe. The project is actively maintained by the community, 
+with more than 50 commits most weeks during 2022, and 20 different developers having made 50 or more 
+commits during the life of the project.
+
+## What exactly is OTP?
+
+The most commonly used OpenTripPlanner component is a server-side Java application that is primarily 
+designed to import map, transit service, and other mobility data sets, input user origins and 
+destinations, and output API endpoints that allow other applications to consume generated trip plans, 
+vector map tiles, stop departures, and other rider information.
+
+In other words, OTP is a backend application that can work with other frontend user interfaces. OTP 
+works with your website, mobile app, physical signage or other applications in order to provide 
+relevant customer information and a meaningful trip plan to riders.
+
+There's no official OTP user interface, although the OpenTripPlanner ecosystem includes several user 
+interface projects. How your OTP looks is entirely up to you.
+
+## What OTP can do for you
+
+### Transit agencies
+
+Transit agencies can use OTP as their public website/app trip planner or as an internal trip planner 
+for their customer service team.
+
+- Fully customizable for use in any frontend
+- High-performance trip planning
+- Fully multimodal routing, including walking, biking, bikeshare, demand-responsive services in 
+combined trip plans rather than presented in alternative trip plans
+- Support for real-time alerts, trip re-routing based on real-time delays
+- Pareto optimal multi-criteria cost calculation allows for dynamic user inputs and requests including 
```

Comment:
"Pareto optimal" jumps into extremely technical detail in an otherwise high-level document. I think the expression "multi-criteria" alone is readily understandable by any reader, while "Pareto optimal" is a detour into specialized economics/mathematics references.